cmd/zoekt-git-clone/main.go

```diff
@@ -32,6 +32,8 @@ import (
 
 func main() {
 	dest := flag.String("dest", "", "destination directory")
+	nameFlag := flag.String("name", "", "name of repository")
+	repoIDFlag := flag.String("repoid", "", "id of repository")
```

Comment:
I was hoping we could use [flag.Uint](https://pkg.go.dev/flag#Uint) here but it creates another tricky situation where we can't easily tell the difference between an unset value and some arbitrarily chosen default value. Since we don't want to add it to `config` unless it's been explicitly set then using a string seemed like the easiest way to do this based on looking at the API of `flag`.

If we wanted to add extra safety we could parse the string as uint32 and then convert it back to string again before putting into `config` but I wasn't sure if this was worth the extra complexity.

Reply:
alternatively you can dissallow negative values / the zero value. Then use that as a signal it wasn't set.